packages/api/test/e2e/ethBridge.e2e.ts

```diff
@@ -280,5 +280,17 @@ describe('Eth bridge test', () => {
 
       done();
     })
+
+    it( 'Get r,s,v from invalid signatuew', async done => {
+      const sign = api.registry.createType('EthereumSignature', '0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000');
```

Comment:
wondering if this works?
```suggestion
      const sign = api.registry.createType('EthereumSignature', null);
```